### PR TITLE
Consistent introduction of ML-KEM-512, various updates

### PR DESCRIPTION
**Pull request by falko-strenzke**
consistent introd. of ML-KEM-512, updates from core pqc draft, Stephan as author, referencing of draft-ietf-openpgp-pqc-03, added introduction, etc.

### CODE REVIEW

File: draft-individual-openpgp-nist-bp-comp.md [hunk 1]
```diff
@@ -276,14 +302,19 @@ informative:
...
 
-This document defines PQ/T composite schemes based on ML-KEM and ML-DSA combined with ECC algorithms for the OpenPGP protocol.
-TODO
+This document defines PQ/T composite schemes based on ML-KEM and ML-DSA combined with ECC algorithms using the NIST and Brainpool domain parameters for the OpenPGP protocol.
+
 --- middle
 
 # Introduction
 
+[draft-ietf-openpgp-pqc-03] introduces post-quantum cryptography in OpenPGP.
```
Comment (TJ-91) [inline draft-individual-openpgp-nist-bp-comp.md:314]:
```suggestion
This document introduces additional PQ/T algorithms in OpenPGP and extends [draft-ietf-openpgp-pqc-03].
```

I just think it's better not to start the sentence / section with a reference

**Review by TJ-91 — APPROVED** [review]
looks good to me

File: draft-individual-openpgp-nist-bp-comp.md (hunk 1 above)
Comment (falko-strenzke) [reply draft-individual-openpgp-nist-bp-comp.md]: I rewrote the start of the introduction accordingly.

**Review by TJ-91 — APPROVED** [review]

**Review by TJ-91 — DISMISSED** [review]

**Review by QuynhDangNIST — CHANGES_REQUESTED** [review]
For NIST, I have: ML-KEM-512 + P-256, ML-KEM-768 + P-384, ML-KEM-1024 + P-384, ML-DSA-44 + NIST-256, ML-DSA-65 + P-384 and ML-DSA-87 + P-384. How do I make the needed changes in this draft ?

**Comment by falko-strenzke** [comment]
> For NIST, I have: ML-KEM-512 + P-256, ML-KEM-768 + P-384, ML-KEM-1024 + P-384, ML-DSA-44 + NIST-256, ML-DSA-65 + P-384 and ML-DSA-87 + P-384. How do I make the needed changes in this draft ?

I have made the requested changes in the last commit 0802fd259fb712f5077a5ca4bae526af153308b0. @QuynhDangNIST Could you re-approve the review please?

**Review by TJ-91 — APPROVED** [review]

**Review by sehlen — APPROVED** [review]

**Pull request merged**